images/scripts/rhel4edge.bootstrap

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -eux
+
+BASE=$(dirname $0)
+
+# Names needs to be unique
+timestamp=$(date +%s)
+request=$(cat <<EOF
+{
+  "name": "rhel4edge-$timestamp",
+  "version": 0,
+  "distribution": "rhel-91",
+  "imageType": "rhel-edge-installer",
+  "packages": [
+    { "name": "cockpit-system" },
+    { "name": "cockpit-bridge" },
+    { "name": "socat" },
+    { "name": "openssh-server" }
+  ],
+  "outputTypes": [
+    "rhel-edge-installer",
+    "rhel-edge-commit"
+  ],
+  "commit": {
+    "arch": "x86_64"
+  },
+  "installer": {
+    "username": "admin",
+    "sshkey": "$(cat $BASE/../../machine/identity.pub)"
+  }
+}
+EOF
+)
+
+# File `/etc/crc_passwd` contains the following line
+# machine console.redhat.com login <username> password <password>
+
+response=$(curl --netrc-file /etc/crc_passwd -X POST https://console.redhat.com/api/edge/v1/images -H "Content-Type: application/json" -d "$request")
```

Comment:
I suggest to use `--json @-` or `--json @request.json` (see above) for correct headers, then you can drop the `-H` and the `-X POST`.

Reply:
done